src/ui/public/chrome/directives/app_switcher/app_switcher.less

```diff
@@ -1,55 +1,168 @@
+@import (reference) "~ui/styles/mixins";
 @import (reference) "~ui/styles/variables";
 
-@app-icon-size: 48px;
-@app-icon-padding: 10px;
+// as - App Switcher
+@as-open-width: 160px;
+@as-closed-width: 53px;
+@app-icon-height: 38px;
+@transition-time: .35s;
+@transition-delay: .25s;
+@app-links-wrapper-background: #3caed2;
 
+@firstLinkColor: #E4BB51;
+@secondLinkColor: #8AC336;
+@thirdLinkColor: #59C6C5;
+
+body { overflow-x: hidden; }
+
+.app-links-wrapper {
+  width: @as-open-width;
+  position: fixed;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 0;
+  background-color: @app-links-wrapper-background;
+  overflow: hidden;
+  transition: width @transition-time;
+  transition-delay: @transition-delay;
+
+  &:hover {
+    .app-title {
+      display: inline-block;
+    }
+    + .app-wrapper {
+      transform: translateX(@as-open-width - @as-closed-width);
+    }
+  }
+
+  .logo {
+    height: 70px;
+    width: @as-open-width;
+    list-style-type: none;
+    &.kibana {
+      background-image: url("~ui/images/kibana.svg");
+      background-position: 6px 10px;
+      background-size: 140px 50px;
+      background-repeat: no-repeat;
+      background-color: #e8488b;
+    }
+  }
+  .bottom-apps {
+    position: absolute;
+    bottom: 0;
+  }
+}
+.app-wrapper {
+  .real-flex-parent();
+  position: absolute;
+  transition: transform @transition-time;
+  transition-delay: @transition-delay;
+  left: @as-closed-width;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+  margin: 0 auto;
+  background-color: #fff;
+
+  &-panel {
+    .flex-parent(@shrink: 0);
+    box-shadow: -4px 0px 3px rgba(0,0,0,0.2);
+  }
+
+  .navbar-right {
+    margin-right: 0;
+  }
+}
 .app-links {
   text-align: justify;
 
   .app-link {
-    display: inline-block;
-    vertical-align: top;
-    text-align: left;
-    width: @app-icon-size + (@app-icon-padding * 2);
-    margin: 0px 10px;
-    padding: @app-icon-padding;
-    border-radius: @border-radius-base;
+    width: @as-open-width;
+    height: @app-icon-height;
+    line-height:  24px;
```

Comment:
The two properties above this are variables, seems like it should be too? Is there a way to base it on `@app-icon-height`?